src/main/java/net/smartcosmos/cluster/userdetails/config/UserAuthenticationProperties.java

```diff
@@ -0,0 +1,28 @@
+package net.smartcosmos.cluster.userdetails.config;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import lombok.Data;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for dummy user.
+ */
+@Data
+@ConfigurationProperties("smartcosmos.security.user")
+public class UserAuthenticationProperties {
+
+    public static final String DEFAULT_NAME = "user";
```

Comment:
to create an "obvious failure" situation let's make this something uncommon, e.g. `changeme-user` or `notconfigured-user`

This is a common tactic I use with default properties to prevent total failure through NPE etc. on startup yet still provide information in the log as to the exact problem.  In this case seeing something obvious for the user in the logs makes it pretty obvious the YML was not configured or not configured correctly.